### PR TITLE
Mention EXTRACT_ALL in the context of /file

### PR DESCRIPTION
**Pull request by moshekaplan**

### CODE REVIEW

File: doc/commands.doc [hunk 1]
```diff
@@ -605,7 +605,8 @@ Structural indicators
 
   \par Important:
   The documentation of global functions, variables, typedefs, and enums will
-  only be included in the output if the file they are in is documented as well.
+  only be included in the output if the file they are in is documented as well
+  or if \ref cfg_javadoc_autobrief "EXTRACT_ALL" is set to \c YES.
```
Comment (albert-github) [inline doc/commands.doc]: The construct ` \ref cfg_javadoc_autobrief "EXTRACT_ALL"` is very strange as the text is `EXTRACT_ALL` but the jump is to `JAVADOC_AUTOBRIEF`.

Is the intention:
  - `\ref cfg_javadoc_autobrief "JAVADOC_AUTOBRIEF"`

or
  - `\ref cfg_extract_all "EXTRACT_ALL"`

or 
  - ...

**Comment by moshekaplan** [comment]
Oops, that should be cfg_extract_all.

**Comment by doxygen** [comment]
I've corrected it.

**Comment by moshekaplan** [comment]
Thank you!

**Pull request merged**